AppCenter/AppCenter/Internals/Channel/MSOneCollectorChannelDelegate.m

```diff
@@ -101,9 +99,7 @@ - (void)channel:(id<MSChannelProtocol>)channel didPrepareLog:(id<MSLog>)log with
   id<MSLogConversion> logConversion = (id<MSLogConversion>)log;
   NSArray<MSCommonSchemaLog *> *commonSchemaLogs = [logConversion toCommonSchemaLogs];
   for (MSCommonSchemaLog *commonSchemaLog in commonSchemaLogs) {
-    dispatch_async(oneCollectorChannelUnit.logsDispatchQueue, ^{
-      [oneCollectorChannelUnit enqueueItem:commonSchemaLog];
-    });
+    [oneCollectorChannelUnit enqueueItem:commonSchemaLog];
```

Comment:
does this do disk access? If yes we shouldn't execute in UI thread.

Reply:
It shouldn't. We execute enqueueItem in the UI thread every time TrackEvent is called.

Reply:
So if we are, a broader change is needed